code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -117,65 +117,81 @@
 /obj/item/reagent_containers/hypospray/autoinjector/antitoxin
 	name = "autoinjector (anti-toxin)"
 	preloaded_reagents = list("anti_toxin" = 5)
+	spawn_blacklisted = TRUE	// These were made for the trade program. Blacklisting until otherwise to prevent unintended consequences.
 
 /obj/item/reagent_containers/hypospray/autoinjector/tricordrazine
 	name = "autoinjector (tricordrazine)"
 	preloaded_reagents = list("tricordrazine" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/kelotane
 	name = "autoinjector (kelotane)"
 	preloaded_reagents = list("kelotane" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/bicaridine
 	name = "autoinjector (bicaridine)"
 	preloaded_reagents = list("bicaridine" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/antirad
 	name = "autoinjector (anti-rad)"
 	preloaded_reagents = list("hyronalin" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/dexalin
 	name = "autoinjector (dexalin)"
 	preloaded_reagents = list("dexalin" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/spaceacillin
 	name = "autoinjector (spaceacillin)"
 	preloaded_reagents = list("spaceacillin" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/tramadol
 	name = "autoinjector (tramadol)"
 	preloaded_reagents = list("tramadol" = 5)
+	spawn_blacklisted = TRUE
 
 // Intermediate
 /obj/item/reagent_containers/hypospray/autoinjector/polystem
 	name = "autoinjector (polystem)"
 	preloaded_reagents = list("polystem" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/meralyne
 	name = "autoinjector (meralyne)"
 	preloaded_reagents = list("meralyne" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/dermaline
 	name = "autoinjector (dermaline)"
 	preloaded_reagents = list("dermaline" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/dexalinplus
 	name = "autoinjector (dexalin plus)"
 	preloaded_reagents = list("dexalinp" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/oxycodone
 	name = "autoinjector (oxycodone)"
 	preloaded_reagents = list("oxycodone" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/ryetalyn
 	name = "autoinjector (ryetalyn)"
 	preloaded_reagents = list("ryetalyn" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/hyperzine
 	name = "autoinjector (hyperzine)"
 	preloaded_reagents = list("hyperzine" = 5)
+	spawn_blacklisted = TRUE
 
 /obj/item/reagent_containers/hypospray/autoinjector/drugs
 	name = "autoinjector (drugs)"
 	preloaded_reagents = list("space_drugs" = 5)
+	spawn_blacklisted = TRUE
```

Comment:
You can add at the top of the file
```
/obj/item/reagent_containers/hypospray/autoinjector
    spawn_blacklisted = TRUE
```
so that all those children type are blacklisted without manually adding it to all of them individually.